tests/integration/integration_steps/fabric_steps.py

```diff
@@ -39,10 +39,13 @@ def write_timeseries_data_to_fabric(credential: DefaultAzureCredential, data_fra
 
 def remove_time_series_data_from_fabric(credential: DefaultAzureCredential, table_path:str):
     token = credential.get_token("https://storage.azure.com/.default").token
-    DeltaTable(
-        table_uri= table_path,
-        storage_options={"bearer_token": token, "use_fabric_endpoint": "true"}
-    ).delete()
+    try:
+        DeltaTable(
+            table_uri= table_path,
+            storage_options={"bearer_token": token, "use_fabric_endpoint": "true"}
+        ).delete()
+    except Exception as e:
```

Comment:
Is there a specific error message you can catch instead? Ignoring Exception is very broad and could potentially hide other errors.